package.js

```diff
@@ -10,7 +10,7 @@ Package.onUse((api) => {
   api.versionsFrom(['1.4', '3.0-beta.0']);
```

Comment:
@vparpoil shall we bump this line for the latest beta too?

Reply:
Hi @dr-dimitru , From what I have seen elsewhere, the practice seems to use '3.0-beta.0' which makes this valid for all `3.0-beta.X` in my understanding 
[Here is an example for collection2](https://github.com/Meteor-Community-Packages/meteor-collection2/blob/master/package/collection2/package.js#L19)

Reply:
@vparpoil perfect 👍